### ISSUE TITLE
standard run database error message is misleading

### ISSUE THREAD

**Issue by tlatorre-uchicago** [snoplus/orca#378]
If you boot up Orca and it can't connect to CouchDB you get the following message in the log:

```
17-04-04 12:04:11 Error reading standard runs from database: The data couldn't be read because it isn't in the correct format.
```

It should probably say that the database is down.

**Comment by fbdescamps** [comment]
@jcaravaca

**Comment by tlatorre-uchicago** [comment]
I think this is fixed in #367. @jcaravaca can you close the issue if it is fixed?

**Comment by jcaravaca** [comment]
Yeah, this should be fine now.
